### PR TITLE
VP-2272 fix instruction for platform v3

### PR DESCRIPTION
**Pull request by AliveMen**
Update instruction according platform v3 requirement

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,6 +104,24 @@ Read more about how to generate API keys [here](https://virtocommerce.com/docs/v
     }
 	...
 ```
+For platform v3 
+```
+...
+"Endpoint": {
+     //Specify platform url
+     "Url": "https://localhost:5001",
+     //Comment the follow settings
```
Comment (yecli) [inline README.md:113]: Comment the following settings

File: README.md [hunk 2]
```diff
@@ -104,6 +104,24 @@ Read more about how to generate API keys [here](https://virtocommerce.com/docs/v
...
+     //"AppId": "...",
+     //"SecretKey":"...",
+
+     //Uncomment the follow settings
```
Comment (yecli) [inline README.md:117]: Uncomment the following settings

File: README.md (hunk 1 above)
Comment (AliveMen) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (AliveMen) [reply README.md]: fixed

**Comment by AliveMen** [comment]
PR to wrong base

**Pull request closed without merging**